Pipfile

```diff
@@ -20,9 +20,12 @@ sphinx = "*"
 sphinxcontrib-django = "*"
 sphinx-last-updated-by-git = "*"
 sphinx-rtd-theme = "*"
+reportlab = "*"
+xhtml2pdf = "*"
```

Comment:
```suggestion
```
I think reportlab can be removed since it is not used and xhtml2pdf can be removed since it is already contained in `setup.py`. If you freshly install the cms, all packages from setup.py get installed automatically and don't need to be contained in the Pipfile.

Reply:
Yes, we also talked about this with @svenseeberg, so i could implement this feature by adding a new action in the footer of the page_tree.html, so that when user sets the checkboxes for all pages, he gets a complete pdf document

Reply:
Referring to your comment: It should also be possible to create a pdf with the whole content of one region. Keep in mind that if there are multiple root pages, you also have multiple tree_ids in one region's pdf.